src/resolvers/Mutation.js

```diff
@@ -192,11 +192,82 @@ async function _createTags(tags_contactNumbers, context) {
   return { create: arr };
 }
 
+async function deletePost(parent, args, context, info) {
+  const id = args.id;
+  return await context.db.mutation.deletePost(
+    {
+      where: {
+        id: id
+      }
+    },
+    info
+  );
+}
+
+async function deleteComment(parent, args, context, info) {
+  const id = args.id;
+  return await context.db.mutation.deleteComment(
+    {
+      where: {
+        id: id
+      }
+    },
+    info
+  );
+}
+
+async function deleteTag(parent, args, context, info) {
+  const id = args.id;
+  return await context.db.mutation.deleteTag(
+    {
+      where: {
+        id: id
+      }
+    },
+    info
+  );
+}
+
+async function deleteUser(parent, args, context, info) {
+  const id = args.id;
```

Comment:
This can only be done by the admin of the group.
So check if the context.userId = to the admin of the ActiveGroup of context.activeGroup